### PR TITLE
Add card component

### PR DESCRIPTION
**Pull request by aesha-spreadd**
Add 5 different kind of card components

1) Simple Card
Screenshot 2023-12-29 at 9 55 02 AM

2) Badge Card
Screenshot 2023-12-29 at 9 55 10 AM

3)Dismiss Card
Screenshot 2023-12-29 at 9 55 20 AM

4) Dismiss Card Overlay
Screenshot 2023-12-29 at 9 55 30 AM

5) Horizontal Card
Screenshot 2023-12-29 at 9 55 40 AM

- [x]  Checking the code is duplicate or not?
- [x]  Is these code has bug
- [ ]  Is code is check by shubhum sir or ankit sir or tara ma'am?
- [ ]  Are you going to merge this or not?

### CODE REVIEW

File: src/Cards/Cards.Styled.js [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+  background-color: ${theme.pastelCardColor};
+`;
+
+export const commonButton = css`
+  background-color: ${theme.textColor};
+  color: ${theme.primary};
+  border: 1px solid ${theme.primary};
+  cursor: pointer;
+  height: 38px;
+  width: 298px;
```
Comment (ankitadrixus08) [inline src/Cards/Cards.Styled.js]: use max width @aeshapatel025

File: src/Cards/Cards.Styled.js [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+  padding: -1px 110px;
```
Comment (ankitadrixus08) [inline src/Cards/Cards.Styled.js]: why in minus padding we don't use this @aeshapatel025

File: src/Cards/Cards.Styled.js [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+  cursor: pointer;
+  border-radius: 4px;
+  font-size: 14px;
+  margin-right: 10px;
+  font-family: Arial;
```
Comment (ankitadrixus08) [inline src/Cards/Cards.Styled.js]: why arial font here in other prs you have different font @aeshapatel025

File: src/Cards/Cards.Styled.js [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+    background-color: ${theme.primary};
+    color: ${theme.textColor};
+  }
+`;
+
+export const CardContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
```
Comment (ankitadrixus08) [inline src/Cards/Cards.Styled.js]: why use vh here @aeshapatel025

File: src/Cards/Cards.Styled.js [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+    color: ${theme.primary};
+    font-size: 18px;
+    font-weight: 500;
+  }
+
+  p {
+    color: ${theme.cardTextColor};
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 19.6px;
```
Comment (ankitadrixus08) [inline src/Cards/Cards.Styled.js]: give roundup value @aeshapatel025

File: src/Cards/Cards.Styled.js [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+export const StyledNormalCard = styled.div`
+  ${commonStyled};
+  display: block;
+  border-radius: 5px;
+  border: 3px solid ${theme.pastelCardColor};
+  column-gap: 10px;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  height: 129.875px;
```
Comment (ankitadrixus08) [inline src/Cards/Cards.Styled.js]: not acceptable @aeshapatel025

File: src/Cards/Cards.Styled.js [hunk 7]
```diff
@@ -0,0 +1,218 @@
...
+  padding: 50px 50px;
+  width: 695px;
```
Comment (ankitadrixus08) [inline src/Cards/Cards.Styled.js]: use max-width @aeshapatel025

File: src/Cards/Cards.Styled.js [hunk 8]
```diff
@@ -0,0 +1,218 @@
...
+  flex-direction: column;
+  height: 109.875px;
```
Comment (ankitadrixus08) [inline src/Cards/Cards.Styled.js]: remove this @aeshapatel025

File: src/Cards/Cards.Styled.js [hunk 9]
```diff
@@ -0,0 +1,218 @@
...
+`;
+
+export const StyledBadgesCard = styled.div`
+  ${commonStyled};
+  position: relative;
+  img {
+    position: relative;
+    object-fit: cover;
+    border-radius: 5px;
+    top: -10px;
```
Comment (ankitadrixus08) [inline src/Cards/Cards.Styled.js]: why in minus @aeshapatel025

File: src/Cards/Cards.Styled.js [hunk 10]
```diff
@@ -0,0 +1,218 @@
...
+
+  div {
+    position: absolute;
+    text-align: center;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    z-index: 2;
+    transition-timing-function: ease;
+    color: rgb(246, 249, 252);
```
Comment (ankitadrixus08) [inline src/Cards/Cards.Styled.js]: why use this rgb @aeshapatel025

File: src/Cards/Cards.Styled.js [hunk 11]
```diff
@@ -0,0 +1,218 @@
...
+  }
+
+  div {
+    position: absolute;
+    text-align: center;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    z-index: 2;
+    transition-timing-function: ease;
```
Comment (ankitadrixus08) [inline src/Cards/Cards.Styled.js]: meaning of this @aeshapatel025

File: src/Cards/Cards.Styled.js [hunk 12]
```diff
@@ -0,0 +1,218 @@
...
+    color: rgb(246, 249, 252);
+    display: block;
+    font-family: Rubik, sans-serif;
+    font-size: 24px;
+    height: 28.5px;
```
Comment (ankitadrixus08) [inline src/Cards/Cards.Styled.js]: remove float value @aeshapatel025

File: src/Cards/Cards.Styled.js [hunk 13]
```diff
@@ -0,0 +1,218 @@
...
+  column-gap: 10px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  font-family: Rubik, sans-serif;
+  font-size: 6px;
+  height: auto;
+  max-width: 700px;
+  padding: 10px;
+  row-gap: 10px;
```
Comment (ankitadrixus08) [inline src/Cards/Cards.Styled.js]: here we can use gap ? @aeshapatel025

File: src/Cards/Cards.Styled.js [hunk 14]
```diff
@@ -0,0 +1,218 @@
...
+  transition: all 0.5s ease;
+
+  h2 {
+    color: rgb(255, 78, 84);
+    display: block;
+    font-family: Rubik, sans-serif;
+    font-size: 18px;
+    font-weight: 500;
+    margin-top: -15px;
```
Comment (ankitadrixus08) [inline src/Cards/Cards.Styled.js]: why this in minus @aeshapatel025

File: src/Cards/Cards.Styled.js [hunk 15]
```diff
@@ -0,0 +1,218 @@
...
+  }
+
+  p {
+    margin-top: -7px;
```
Comment (ankitadrixus08) [inline src/Cards/Cards.Styled.js]: not needed here in minus @aeshapatel025

Comment (aesha-spreadd) [reply src/Cards/Cards.Styled.js]: yes ! i have remove it

File: src/Cards/Cards.Styled.js (hunk 14 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.Styled.js]: to add the h2 title there but now i have removed it and change that

File: src/Cards/Cards.Styled.js (hunk 13 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.Styled.js]: yes! i have remove them and add gap

File: src/Cards/Cards.Styled.js (hunk 12 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.Styled.js]: changes done!

File: src/Cards/Cards.Styled.js (hunk 8 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.Styled.js]: done changes

File: src/Cards/Cards.Styled.js (hunk 7 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.Styled.js]: done changes !

File: src/Cards/Cards.Styled.js (hunk 6 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.Styled.js]: done changes

File: src/Cards/Cards.Styled.js (hunk 5 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.Styled.js]: done changes!

File: src/Cards/Cards.Styled.js (hunk 1 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.Styled.js]: done changes

File: src/Cards/Cards.Styled.js (hunk 2 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.Styled.js]: changes done

File: src/Cards/Cards.Styled.js (hunk 3 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.Styled.js]: changes done

File: src/Cards/Cards.Styled.js (hunk 4 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.Styled.js]: i have change it into px

File: src/Cards/Cards.Styled.js [hunk 16]
```diff
@@ -0,0 +1,218 @@
+import styled, { css } from 'styled-components';
+import theme from '../theme';
+
+export const commonStyled = css`
```
Comment (tarakhant1297) [inline src/Cards/Cards.Styled.js]: @aeshapatel025 add prefix styled to styled component

File: src/Cards/Cards.Styled.js [hunk 17]
```diff
@@ -0,0 +1,218 @@
...
+  margin: 20px;
+  border-radius: 5px;
+  height: auto;
+  padding: 10px 10px;
+  width: 300px;
+  background-color: ${theme.pastelCardColor};
+`;
+
+export const commonButton = css`
```
Comment (tarakhant1297) [inline src/Cards/Cards.Styled.js]: @aeshapatel025 same as above

File: src/Cards/Cards.Styled.js [hunk 18]
```diff
@@ -0,0 +1,218 @@
...
+  text-transform: uppercase;
+
+  &:hover,
+  &:active {
+    background-color: ${theme.primary};
+    color: ${theme.textColor};
+  }
+`;
+
+export const CardContainer = styled.div`
```
Comment (tarakhant1297) [inline src/Cards/Cards.Styled.js]: @aeshapatel025 add prefix styled

File: src/Cards/Cards.js [hunk 19]
```diff
@@ -0,0 +1,24 @@
...
+  title,
+  text,
+  textUnique,
+  unique,
+  imgSrc,
+  labelText,
+  children,
+}) => {
+  return (
+    <StyledCardComponent unique={unique} textUnique={textUnique}>
```
Comment (tarakhant1297) [inline src/Cards/Cards.js]: @aeshapatel025why you pass `unique` and `textUnique ` props

File: src/Cards/Cards.js [hunk 20]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { StyledCardComponent, labelText as LabelWrapper } from './Cards.Styled';
+
+const Cards = ({
+  title,
+  text,
+  textUnique,
+  unique,
+  imgSrc,
+  labelText,
```
Comment (tarakhant1297) [inline src/Cards/Cards.js]: @aeshapatel025 give proper props name to card

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]
https://ant.design/components/card follow this for card props

File: src/Cards/Cards.js (hunk 20 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.js]: Changes done!

File: src/Cards/Cards.js (hunk 19 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.js]: Changes done!

File: src/Cards/Cards.Styled.js (hunk 18 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.Styled.js]: changes done!

File: src/Cards/Cards.Styled.js (hunk 17 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.Styled.js]: changes done!

File: src/Cards/Cards.Styled.js (hunk 16 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.Styled.js]: changes done

File: src/Cards/Cards.Styled.js (hunk 9 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.Styled.js]: to give it a proper position but done with it

File: src/Cards/Cards.Styled.js [hunk 21]
```diff
@@ -0,0 +1,189 @@
...
+export const StyledCardComponent = styled.div`
+  font-family: Rubik, sans-serif;
+  h2 {
+    color: ${theme.primary};
+    font-size: 18px;
+    font-weight: 500;
+  }
+
+  p {
+    color: ${theme.cardTextColor};
```
Comment (tarakhant1297) [inline src/Cards/Cards.Styled.js]: @aeshapatel025 give proper name to `cardTextColor `

File: src/Cards/Cards.Styled.js [hunk 22]
```diff
@@ -0,0 +1,189 @@
...
+  width: 100%;
+  max-width: 300px;
+`;
+
+export const label = styled.div`
+  position: absolute;
+  top: 30px;
+  left: 0px;
+  background-color: ${theme.primary};
+  color: ${theme.textColor};
```
Comment (tarakhant1297) [inline src/Cards/Cards.Styled.js]: @aeshapatel025 give proper name to textColor

File: src/Input/Input.Styled.js [hunk 23]
```diff
@@ -0,0 +1,34 @@
...
+
+  label {
+    display: block;
+    margin: 0 0 8px 0;
+    font-size: 15px;
+    color: ${(props) => (props.color ? theme.primary : 'black')};
+  }
+`;
+
+export const Input_Container_inputs_type = styled.div`
```
Comment (tarakhant1297) [inline src/Input/Input.Styled.js]: @aeshapatel025 give proper name  to `Input_Container_inputs_type `

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

File: src/Input/Input.Styled.js (hunk 23 above)
Comment (aesha-spreadd) [reply src/Input/Input.Styled.js]: changes done!

File: src/Cards/Cards.Styled.js (hunk 22 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.Styled.js]: changes done!

File: src/Cards/Cards.Styled.js [hunk 24]
```diff
@@ -0,0 +1,189 @@
...
+
+export const StyledNormalCard = styled.div`
+  ${StyledCommon};
+  display: block;
+  border: 3px solid ${theme.pastelColor};
+  width: 100%;
+  max-width: 300px;
+`;
+
+export const label = styled.div`
```
Comment (tarakhant1297) [inline src/Cards/Cards.Styled.js]: @aeshapatel025 add styled prefix

File: src/Cards/Cards.Styled.js [hunk 25]
```diff
@@ -0,0 +1,189 @@
...
+  top: 30px;
+  left: 0px;
+  background-color: ${theme.primary};
+  color: ${theme.neutral};
+  font-weight: 600;
+  font-size: 14px;
+  padding: 5px 10px;
+`;
+
+export const closeIcon = styled.div`
```
Comment (tarakhant1297) [inline src/Cards/Cards.Styled.js]: @aeshapatel025 add styled prefix

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

File: src/Cards/Cards.Styled.js (hunk 25 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.Styled.js]: changes done

File: src/Cards/Cards.Styled.js (hunk 24 above)
Comment (aesha-spreadd) [reply src/Cards/Cards.Styled.js]: changes done

**Review by tarakhant1297 — APPROVED** [review]

**Comment by aesha-spreadd** [comment]
Add Card components

**Pull request merged**